samples/hosting/docker/Core_7/Receiver/Program.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+class Program
+{
+    static async Task Main(string[] args)
+    {
+        //required to prevent possible occurrence of .NET Core issue https://github.com/dotnet/coreclr/issues/12668
+        Thread.CurrentThread.CurrentUICulture = new CultureInfo("en-US");
+        Thread.CurrentThread.CurrentCulture = new CultureInfo("en-US");
+
+        Console.CancelKeyPress += new ConsoleCancelEventHandler(OnExit);
+
+        Console.Title = "Samples.Docker.Receiver";
+        LogManager.Use<DefaultFactory>()
+            .Level(LogLevel.Info);
+
+        var endpointInstance = await TryStart()
+            .ConfigureAwait(false);
+        
+        Console.WriteLine("Press Ctrl+C to exit.");
```

Comment:
why `Ctrl+C` instead of any key like all the other samples?

Reply:
Within Docker the input is redirected, no terminal with keyboard input is available. We're really handling a signal here (SIGINT, IIRC). For the user running this interactively will get the same effect when Ctrl+C is used.

Reply:
just an FYI.  Console.CancelKeyPress has been broken when debugging in VS for years

https://connect.microsoft.com/VisualStudio/feedback/details/524889/debugging-c-console-application-that-handles-console-cancelkeypress-is-broken-in-net-4-0

hence none of the samples use it

Reply:
Thanks. I would have preferred use `Console.ReadKey()` like all the other samples but that emits warnings about terminal redirections when running in a container.